src/main/java/org/jabref/model/entry/Date.java

```diff
@@ -44,7 +44,17 @@ public class Date {
                 "uuuu.M.d",                             // covers 2015.1.15
                 "uuuu",                                 // covers 2015
                 "MMM, uuuu",                            // covers Jan, 2020
-                "uuuu.MM.d"                             // covers 2015.10.15
+                "uuuu.MM.d",                            // covers 2015.10.15
+                "y-",                                   // covers 2015-
+                "-y",                                   // covers -2015
+                "y-M/y-M",                              // covers 2015-01/2015-02
+                "MMMM y/MMMM y",                        // covers January 2015/February 2015
+                "d MMMM y/d MMMM y",                    // covers 20 January 2015/20 February 2015
+                "y G",                                  // covers 1 BC
+                "y G / y G",                            // covers 30 BC / 5 AD
+                "yyyy G / yyyy G",                      // covers 0030 BC / 0005 AD
+                "yyyy-MM G / yyyy-MM G",                // covers 0030-01 BC / 0005-02 AD
```

Comment:
These date ranges are not yet parsed correctly. The static parse method must probably be adapted somehow.

Reply:
Yes, see https://docs.oracle.com/en/java/javase/19/docs/api/java.base/java/time/format/DateTimeFormatter.html
and https://github.com/JabRef/jabref/blob/170033a37cd1c6b80ef934428ff1e5140f986cda/src/main/java/org/jabref/model/entry/Date.java#L102-L103

There are already existing unit tests which you need to extend.